gis/shapefile_geojson_shapefile/batch_convert_shapefile_to_geojson.py

```diff
@@ -0,0 +1,79 @@
+import os
+import sys
+from glob import glob
+
+# import gui tkinter
+from tkinter import Tk
+import tkinter.filedialog as tkfd
+
+import shapefile
```

Comment:
More of a question: why should we use here `shapefile` package and not the fiona as we did in the other utilities? Would it help in consistency to use a similar reading approach?